R/derive_extreme_records.R

```diff
@@ -62,7 +67,7 @@
 #'   For all other by groups `exist_flag` is set to the value specified by the
 #'   `false_value` argument.
 #'
-#'   *Permitted Values:* Variable name
+#' @permitted Variable name
 #'
 #' @param true_value True value
 #'
```

Comment:
Mention in the details below also that if `dataset` is not passed, a new dataset is created.

Reply:
Done